contracts/verifiers/UniversalVerifier.sol

```diff
@@ -21,216 +23,234 @@ contract UniversalVerifier is
     }
 
-    /// @dev Disables ZKP Request
-    /// @param requestId The ID of the ZKP request
-    function disableZKPRequest(uint64 requestId) public onlyOwnerOrRequestOwner(requestId) {
-        _disableZKPRequest(requestId);
+    /**
+     * @dev Disables Request
+     * @param requestId The ID of the request
+     */
+    function disableRequest(uint256 requestId) public onlyOwnerOrRequestOwner(requestId) {
+        _disableRequest(requestId);
     }
 
-    /// @dev Enables ZKP Request
-    /// @param requestId The ID of the ZKP request
-    function enableZKPRequest(uint64 requestId) public onlyOwnerOrRequestOwner(requestId) {
-        _enableZKPRequest(requestId);
+    /**
+     * @dev Enables Request
+     * @param requestId The ID of the request
+     */
+    function enableRequest(uint256 requestId) public onlyOwnerOrRequestOwner(requestId) {
+        _enableRequest(requestId);
     }
 
-    /// @dev Add new validator to the whitelist
-    /// @param validator Validator address
-    function addValidatorToWhitelist(ICircuitValidator validator) public onlyOwner {
+    /**
+     * @dev Adds a validator to the whitelist
+     * @param validator The address of the validator
+     */
+    function addValidatorToWhitelist(IRequestValidator validator) public onlyOwner {
         _addValidatorToWhitelist(validator);
     }
 
-    /// @dev Remove validator from the whitelist
-    /// @param validator Validator address
-    function removeValidatorFromWhitelist(ICircuitValidator validator) public onlyOwner {
+    /**
+     * @dev Removes a validator from the whitelist
+     * @param validator The address of the validator
+     */
+    function removeValidatorFromWhitelist(IRequestValidator validator) public onlyOwner {
         _removeValidatorFromWhitelist(validator);
     }
 
+    /**
+     * @dev Disables an auth method
+     * @param authMethod The auth method to disable
+     */
+    function disableAuthMethod(string calldata authMethod) public override onlyOwner {
+        super.disableAuthMethod(authMethod);
+    }
+
+    /**
+     * @dev Enables an auth type
+     * @param authMethod The auth type to enable
+     */
+    function enableAuthMethod(string calldata authMethod) public override onlyOwner {
+        super.enableAuthMethod(authMethod);
+    }
+
     function _getRequestIfCanBeVerified(
-        uint64 requestId
+        uint256 requestId
     )
         internal
         view
-        override(RequestDisableable, ValidatorWhitelist, ZKPVerifierBase)
-        onlyEnabledRequest(requestId)
-        returns (IZKPVerifier.ZKPRequest storage)
+        override(RequestDisableable, ValidatorWhitelist, Verifier)
+        returns (IVerifier.RequestData storage)
     {
         return super._getRequestIfCanBeVerified(requestId);
     }
+
+    function _setRequest(
+        Request calldata request
+    ) internal virtual override(RequestOwnership, ValidatorWhitelist, Verifier) {
+        super._setRequest(request);
```

Comment:
Does it check RequestOwnership, ValidatorWhitelist?

Reply:
It creates request owner because at `RequestOwnership` level and check if validator is whitelisted at `ValidatorWhitelist` level.